tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/polyparser/DependencyParsingTransitionSystem.scala

```diff
@@ -160,6 +160,26 @@ object DependencyParsingTransitionSystem {
     }
     )
   }
+
+  /** Since the DependencyParsingTransitionSystem does POS tagging and arc labeling simultaneously
+    * using a single compound label, this is a convenience function to retrieve the actual
+    * arc label's "label" component.
+    */
+  def getArcLabelSymbol(arcLabel: ArcLabel): Symbol = {
+    arcLabel match {
+      case DependencyParsingArcLabel(stanLabel, _) => stanLabel
+    }
+  }
+
+  /** Since the DependencyParsingTransitionSystem does POS tagging and arc labeling simultaneously
+    * using a single compound label, this is a convenience function to retrieve the actual
+    * arc label's coarse POS component.
+    */
+  def getArcLabelCpos(arcLabel: ArcLabel): Symbol = {
+    arcLabel match {
+      case DependencyParsingArcLabel(_, cpos) => cpos
+    }
+  }
```

Comment:
What happens, here and above, when the `arcLabel` isn't an instance of `DependencyParsingArcLabel`?  Will that ever happen?  Do we need to worry about it?